Library/Formula/khal.rb

```diff
@@ -89,14 +89,15 @@ class Khal < Formula
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    version = Language::Python.major_minor_version "python"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
```

Comment:
[This sort of thing](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/z3.rb).